src/control/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+add_library(control STATIC
+        ControlUtils.cpp
+        BasicPosControl.cpp
+        PosController.cpp
+        numtrees/NumTreePosControl.cpp
+        numtrees/PathPoint.cpp
+        numtrees/Collision.cpp
+        shot-controllers/ShotController.cpp
+        ball-handling/BallHandlePosControl.cpp
+        ball-handling/DribbleBackwards.cpp
+        ball-handling/DribbleForwards.cpp
+        ball-handling/RotateAroundBall.cpp
+        ball-handling/RotateWithBall.cpp
+        )
+
+find_package(Qt5Charts REQUIRED)
+find_package(Qt5Widgets REQUIRED)
+find_package(Qt5Gui REQUIRED)
+
+target_include_directories(control
+        PUBLIC ../../include/roboteam_ai
+        PUBLIC ../..
```

Comment:
Yeah.... If you're not going to refactor the directory inclusion and linking then you're not really solving a problem at all, just creating more by making it a library.

Reply:
True, will take a look at it tomorrow :)